[IMP] project: documentation overhaul (#581)
odoo/documentation-user: project/tasks/email_alias.rst

==========================================
Create Project's Tasks from an Email Alias
==========================================

> When you already have an email address that customers know from the top of their heads, changing
> it is the last think you want to do. Instead, simply link that address to your project and transform
> those conversations into structured work. It automatically creates a task in the first stage
> of a project.

Reviewer: ```suggestion
When you already have an email address that customers know from the top of their heads, changing
it is the last thing you want to do. Instead, link that address to your project and transform
those conversations into structured work. It automatically creates a task in the first stage
of a project.
```

Set up an incoming email server
===============================

> | On the *Settings* application enable *External Email Servers* and define the incoming email
>   alias you would like to use.
> | **For more information**: :doc:`../../discuss/email_servers`

Reviewer: ```suggestion
| On the *Settings* application, enable *External Email Servers* and define the incoming email
  alias you would like to use.
| **For more information**: :doc:`../../discuss/email_servers`
```

Configure the email alias in your project
-----------------------------------------

> | Now that you have the incoming email server set up, go to
>   :menuselection:`Project --> Configuration --> Projects --> Edit`. Under the *Emails* tab define
>   the wanted email alias and choose the policy to receive a message.
> | In addition, you can now also directly set it when creating a new project.

Reviewer: ```suggestion
| Now that you have the incoming email server set up, go to
  :menuselection:`Project --> Configuration --> Projects --> Edit`. Under the *Emails* tab, define the wanted email alias and choose the policy to receive a message.
| In addition, you can now directly set it when creating a new project.
```

.. image:: media/email_alias.png
   :align: center

>    :alt: In the settings of your project define the emails alias under the tab email in Odoo Project

Reviewer: ```suggestion
   :alt: In the settings of your project, define the emails alias under the tab email in Odoo Project
```

.. important::
   All the recipients of the email (To/Cc/Bcc) are automatically added as followers of the task.

The email can be seen under the name of your project on the dashboard.

.. image:: media/email_test.png
   :align: center
   :height: 300
   :alt: View of the email alias chosen on the dashboard view in Odoo Project

.. seealso::

>    - :doc:`./get_started`
>    - :doc:`./collaborate`

Reviewer: ```suggestion
   - :doc:`../get_started`
   - :doc:`../collaborate`
```

Author: hum.. no, I believe I'm right with one . because the files I want to redirect people to are in the same folder.

Reviewer: Ok, my bad ^^
